index.html

```diff
@@ -0,0 +1,195 @@
+        <li><a href="about.html">About</a></li>
+        <li><a href="#partners">Sponsor</a></li>
+        <li><a href="about.html">News</a></li>
+        <li><a class="webinar" href="#">Eneregy</a></li>
+      </ul>
+    </nav>
+    <section class="containerOne">
+      <button class="hamburger" type="button">
+        <i class="menuIcon material-icons">menu</i>
+        <i class="closeIcon material-icons">close</i>
+      </button>
+      <ul class="menu">
+        <li><a class="menuItem" href="index.html">Home</a></li>
+        <li><a class="menuItem" href="#programs">Program</a></li>
+        <li><a class="menuItem" href="about.html">About</a></li>
+        <li><a class="menuItem" href="#partners">Sponsor</a></li>
+        <li><a class="menuItem" href="about.html">News</a></li>
+      </ul>
+
+      <div class="imageDiv">
+        <h3>
+          "Enhancing Green Energy, <br />
+          We are Informing you to protect the enviroment <br/>via adopting Green Energy"
+        </h3>
+        <h1>
+          Eneregy-View: Green Eneregy roles<br />
+          to optimize global warming <br />
+          all over the world.
+        </h1>
+        <p class="textBox">
+          Green energy is any energy type that is generated from natural resources, 
+          such as sunlight, wind or water. It often comes from renewable energy 
+          sources although there are some differences between renewable and green energy.<br/><br/>
+          <b>How Does it Work?</b><br/>
+          As a source of energy, green energy often comes from renewable energy technologies such as solar energy, wind power, geothermal energy, biomass and hydroelectric power. Each of these technologies works in different ways, whether that is by taking power from the sun, as with solar panels, or using wind turbines or the flow of water to generate energy.
+          <br/><br/>
+          </p>
+        <time datetime="2022-11-23">2022.01.11(Wed) ~ 35(Fri)</time>
+        <h5>@ Green Energy Main Officer, HAB ASM, Ethiopia</h5>
+      </div>
+    </section>
+    <section class="programs" id="programs">
+      <h2>Main Program</h2>
+      <hr class="programLine" />
+      <div class="progContainer">
+        <div class="programCards">
+          <i class="fa fa-users fa-2x programSocial"></i>
+          <div><h3>Lecture</h3></div>
+          <p>
+            Four key climate change indicators – greenhouse gas concentrations, 
+            sea level rise, ocean heat and ocean acidification
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-rocket fa-2x programSocial"></i>
+          <div><h3>Exhibition</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-comments-o fa-2x programSocial"></i>
+          <div><h3>Forum</h3></div>
```

Comment:
- [ ] please arrange your sections well in the mobile version like in the original design
yours
![Screenshot (617)_LI](https://user-images.githubusercontent.com/65298048/203501535-b8a29131-c213-4367-9e3a-fade741f6c7d.jpg)

behance

![Screenshot (616)](https://user-images.githubusercontent.com/65298048/203501607-66611127-43a3-4e11-ae5c-ec71684b53f0.png)